README.md

```diff
@@ -0,0 +1,370 @@
+gcry_sha256 gcry_twofish gcry_whirlpool gfxmenu gfxterm 
+gzio halt http increment iso9660 jpeg loadenv loopback
+linux lvm luks luks2 mdraid09 mdraid1x minicmd net 
+normal part_apple part_msdos part_gpt password_pbkdf2
+pgp png reboot regexp search search_fs_uuid search_fs_file
+search_label serial sleep syslinuxcfg test tftp version
+video xfs zstd 
+```
+*******************************************************************************
+### If you are using systemd-boot on arm64 or riscv, is the fix for [unverified Devicetree Blob loading](https://github.com/systemd/systemd/security/advisories/GHSA-6m6p-rjcq-334c) included?
+*******************************************************************************
+N/a
+
+*******************************************************************************
+### What is the origin and full version number of your bootloader (GRUB2 or systemd-boot or other)?
+*******************************************************************************
+We use the same version as Rocky - Grub 2.06-92
+
+*******************************************************************************
+### If your SHIM launches any other components, please provide further details on what is launched.
+*******************************************************************************
+We have successfully packaged and tested a RockyLinux version of certwrapper (formerly certmule).  That is, a certmule package signed by us, but containing the Rocky Linux CA.
+
+This seems perfect for our use-case, as the Rocky grub2 + fwupd upstream packages could be used as-is without the need for recompilation or re-signing.  While keenly interested in kernel modifications, we don't have as much cause to update fwupd or grub2, and would prefer to use our upstream whenever feasible.
+
+I want to inquire about signing this wrapper efi and making it available to users.
+
+The certmule package in question (with the embedded Rocky CA) is located at:  https://github.com/ctrliq/certmule-rocky
+
+*******************************************************************************
+### If your GRUB2 or systemd-boot launches any other binaries that are not the Linux kernel in SecureBoot mode, please provide further details on what is launched and how it enforces Secureboot lockdown.
+*******************************************************************************
+No, Linux kernel launches are all we are interested in.
+
+
+*******************************************************************************
+### How do the launched components prevent execution of unauthenticated code?
+*******************************************************************************
+In the case of the kernel, both the RHEL variant and the upstream ("new") variants prevent this by default, and we do not want to change that.
+
+In the case of Grub + Fwupd, we will be running the same Rocky/RHEL versions unmodified, which also do not execute unauthenticated code by default.
```

Comment:
Didn't we have to modify fwupd? Or was it just a rebuild.

Reply:
It was a rebuild from upstream for RL 9.